### ISSUE TITLE
(R3-004) - (ALL) - Email Attachment Issue

### ISSUE THREAD

**Issue by altrdgenetics** [CurrentlyNotSet/SERB#571]
The attached file would not docket into the system. The ebody is imported but the document is missing. It has special characters.

Attachment Filename:
UNION’S RESPONSE TO EMPLOYER’S SUR-SURREPLY 6.14.17.pdf

**Comment by altrdgenetics** [comment]
**TO JUSTIN:**

> Justin,
> 
> Please provide the steps to recreate this issue
> 
> - Andrew

**Comment by altrdgenetics** [comment]
**FROM JUSTIN:**

Below is the additional text from Erin Conn…
This email was received in the MED box.  The email imported into docketing but the attachment did not.  I deleted it out and recopied to the “to be filed” folder, but the attachment still did not come over.  I checked the SMDS email application- there were errors.  As a fix I just saved the attachment to the scan folder and we were able to docket it that way. Case 2015-MED-02-0104.
 
I do not have the errors as this was when I was out of the office.
Thank you,
 
Justin Brown
